twitter-covid/age-spikes-discussion/src/main/scala/Main.scala

```diff
@@ -0,0 +1,14 @@
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import EUSpikes._
+object Main {
+  def main(args: Array[String]): Unit = {
+    val spark = SparkSession.builder()
+      .appName("EUSpikes")
+      .master("yarn")
```

Comment:
might want to comment that to run this locally you should do "*" or .master("local[4]")